src/DotNetInstallSdk/InstallerLauncher.cs

```diff
@@ -9,15 +11,34 @@ public interface IInstallerLauncher
 
     public class InstallerLauncher : IInstallerLauncher
     {
+        readonly ITextWriter _writer;
+
+        public InstallerLauncher(ITextWriter writer)
+        {
+            _writer = writer;
+        }
+        
         public void Launch(string installerPath)
         {
-            var processStartInfo = new ProcessStartInfo
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
             {
-                FileName = installerPath,
-                UseShellExecute = true
-            };
+                _writer.WriteLine($"The SDK has been downloaded to: {installerPath}");
+                return;
+            }
 
-            Process.Start(processStartInfo)?.WaitForExit();
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                Run("sudo", $"installer -pkg {installerPath} -target /");
+                return;
+            }
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Run(installerPath, $"/install /quiet /norestart");
```

Comment:
What are your thoughts around making this a flag, either a flag to use headless installs, or a flag to disable a headless install? For instance:

`$ dotnet install-sdk --headless true`

Reply:
I'd imagine the majority of the time people would prefer for the install to be headless.

Reply:
That's a good call, and I agree that the default should probably be silent, even though this changes the current behaviour.

Reply:
Yeah, we can release it as a new version number. Better to get these kind of changes ironed out early on.